docs/scripts/app.js

```diff
@@ -18,7 +18,7 @@ for (var i = 0; i < links.length; i++) {
     var linkFile = linkParts[linkParts.length - 1]
 
     if (urlFile === linkFile) {
-        el.className = 'active'
+        el.className += 'active'
```

Comment:
this fixes the mobile navbar which would never show on ol-kit.com b/c it wiped all other classes